pkg/deploy/deploy.go

```diff
@@ -284,13 +284,17 @@ func (c *componentDeployer) removeChildren(ctx context.Context, parent, root gra
 		}
 		childCfg := child.Config
 
+		l := log.WithCtxFields(ctx).WithFields(
+			field.F("parent", parent.Config.Coordinate),
+			field.F("deploymentFailed", failed),
+			field.F("child", child.Config.Coordinate))
+
 		// after the first iteration
 		if parent != root {
-			log.WithCtxFields(ctx).WithFields(field.F("parent", parent.Config.Coordinate), field.F("deploymentFailed", failed), field.F("root", root.Config.Coordinate)).
+			l.WithFields(field.F("root", root.Config.Coordinate)).
```

Comment:
Isn't the `root` anyway always the coordinate the log is for?
At least per your internal message I understood that as the reason to keep the coordinate fixed to that of the config that's the source of things being skipped. 
So IMO it's not really necessary to add it another time